Makefile.am

```diff
@@ -996,7 +996,8 @@ netdata_LDADD = \
 
 if ACLK_NG
     netdata_LDADD += $(OPTIONAL_PROTOBUF_LIBS) \
```

Comment:
Ideally, we'd add the protobuf libs at the right place in the common libs variable.